core/templates/services/math-interactions.service.ts

```diff
@@ -60,10 +60,21 @@ export class MathInteractionsService {
       errorMessage = 'Your answer includes a division by zero, which is ' +
         'not valid.';
     }
-    if (errorMessage.indexOf('is not a valid postfix operator.') !== -1) {
+    if (errorMessage.indexOf('is not a valid postfix operator.') !== -1 &&
```

Comment:
I think you can simplify the logic quite a bit since these if's are connected. In general try not to repeat code unnecessarily:

```
if (errorMessage.indexOf('...') !== -1) {
  let humanReadableOperator = (
    errorMessage[0] === '*' ? '×' :
    errorMessage[0] === '/' ? '÷' :
    errorMessage[0]);
 
 errorMessage = (
   ...); // Construct the error message here.
}
```